vizro-core/src/vizro/models/_components/tabs.py

```diff
@@ -25,6 +25,12 @@ class Tabs(VizroBaseModel):
     # TODO[mypy], see: https://github.com/pydantic/pydantic/issues/156 for tabs field
     tabs: conlist(Container, min_length=1)  # type: ignore[valid-type]
 
+    @field_validator("tabs")
+    def validate_tabs_have_titles(cls, tabs: list[Container]) -> list[Container]:
+        if any(tab.title == "" for tab in tabs):
+            raise ValueError("Each `Container` in `tabs` requires a `title` provided.")
```

Comment:
```suggestion
            raise ValueError("Each `Container` in `tabs` requires an explicit `title`.")
```
Maybe one for the native speakers - found the above sounded weird. We should also stress that it needs to be explicitly set, because it already does have a default

Reply:
See above for my suggested wording :)